jenkins-slaves/ansible/data/config/bashrc

```diff
@@ -6,7 +6,7 @@ if [ -f /etc/bashrc ]; then
 fi
 
 export LANG=en_US.UTF-8
-export JAVA_HOME=/opt/tools/jdk1.8
+export JAVA_HOME=/opt/tools/jdk11
```

Comment:
Not sure if this could break product builds, but I think we explicitely set JDK in the Jenkins jobs, right? @mbiarnes @mareknovotny

Reply:
Please revert this and just use `kie-jdk11` tool from your pipelines. JAVA_HOME will be set to `/opt/tools/jdk11`

Reply:
Reverted back to jdk8